.github/workflows/onPRClose.yml

```diff
@@ -24,10 +26,12 @@ jobs:
   destroy:
     name: Call Azure Pipeline
     runs-on: ubuntu-latest
+    needs: tagOnPRMerge
```

Comment:
super tiny in the grand scheme of things...

If you need the issue number extracted from `tagOnPRMerge` it migth be better to split it into 3 jobs (extract issue number, apply staging label, destory pipeline), where the last 2 needs the first one.

as it stands i think it'll need to be performed serially and you may as well add this as a step in the same job, and if for any reason if the staging label step failed it probably won't run the destroy pipeline

Reply:
made those changes